action.yml

```
@@ -0,0 +1,36 @@
+name: 'Hello World'
+description: 'Greet someone'
+inputs:
+  sources:
+    description: 'One or more directories / files to upload '
+    required: true
+    default: '.'
+  artifacts_url:
+    description: 'artifacts url'
+    required: true
+  username:
+    description: 'artifacts username'
+    required: false
+  password:
+    description: 'artifacts password'
+    required: false
```

For all those vars, we said that we were gonna set them as optional, and the default would be the secret. Can we put those secret in the action itself? Or was there something limiting you in some way?

```suggestion
  artifacts_url:
    description: 'artifacts url'
    required: false
    default: ${{ secrets.ARTIFACTS_URL }}
  username:
    description: 'artifacts username'
    required: false
    default: ${{ secrets.ARTIFACTS_USER }}
  password:
    description: 'artifacts password'
    required: false
    default: ${{ secrets.ARTIFACTS_PASSWORD }}
```

---

hopefully the action don't have access to repository secrets, so we can't set default like var like this.